src/app/shared/components/header/header.component.html

```diff
@@ -0,0 +1,93 @@
+<header class="header">
+  <mat-toolbar class="header__toolbar">
+    <a class="header__logo-wrapper" routerLink="/" matTooltip="QuillPost">
+      <div class="header__logo">
+        <img
+          ngSrc="assets/resources/vectors/logo.svg"
+          alt="QuillPost"
+          width="40"
+          height="40"
+          priority />
+      </div>
+      <span class="header__title">QuillPost</span>
+    </a>
+    <span class="header__spacer"></span>
+    @if (isLoggedIn) {
+      <ng-container *ngTemplateOutlet="navLinks"></ng-container>
+    }
+    @if (!isLoggedIn && !isAuthRoute) {
+      <ng-container *ngTemplateOutlet="authLinks"></ng-container>
+    }
+  </mat-toolbar>
+  <mat-divider></mat-divider>
+</header>
+
+<ng-template #navLinks>
+  <a
+    class="header__create"
+    mat-button
+    matTooltip="Write"
```

Comment:
```suggestion
     mat-button
    class="header__create"
    matTooltip="Write"
```

Check at other places also